sdk/src/entrypoint_native.rs

```diff
@@ -146,6 +146,8 @@ macro_rules! declare_loader(
     )
 );
 
+pub type ProcessInstruction = fn(&Pubkey, &[KeyedAccount], &[u8]) -> Result<(), InstructionError>;
```

Comment:
This mimics the ProcessInstruction in sdk/src/entrypoint.rs so I felt pretty good about promoting this type from runtime/ to sdk/

Reply:
This file should not be part of our developer sdk anyway since it's native program specific